articles/synapse-analytics/backuprestore/restore-sql-pool-from-deleted-workspace.md

```diff
@@ -50,11 +50,11 @@ In this article, you learn how to restore a dedicated SQL pool in Azure Synapse
     Connect-AzAccount
     Set-AzContext -SubscriptionID $SubscriptionID
     
-    # Define the approximate point in time the workspace was dropped as DroppedDateTime "yyyy-MM-ddThh:mm:ssZ" (ex. 2022-01-01T16:15:00Z)
-    $PointInTime="<DroppedDateTime>" 
-    $DroppedDateTime = Get-Date -Date $PointInTime 
-    
-    
+    # Get the exact date and time the workspace SQL pool was dropped. Please do not provide an approximate value. Use the below script to get the exact date time value. 
+    Get-AzSynapseSqlPoolRestorePoint -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -Name $SQLPoolName
+    # Pick the desired restore point after verifying the date time output from the above command. Add it to the where predicate in the below script where x/xx/xxxx is seen.
+    $PointInTime = Get-AzSynapseSqlPoolRestorePoint -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -Name $SQLPoolName | Where-Object "RestorePointCreationDate" -eq "x/xx/xxxx x:xx:xx xM" | Select-Object -ExpandProperty RestorePointCreationDate
+  
```

Comment:
@azurepocmain 
This appears to change the $pointInTime variable from the dropped date of the workspace, to the latest restore point time. That right?
The later script sample uses `$DroppedDateTime` variable as well. Please update either this code sample or the later one.

Reply:
By specifying "x/xx/xxxx x:xx:xx xM" we are still asking customers to bullseye the exact point in time the workspace was dropped. Have we solved the problem?

Reply:
Thanks @WilliamDAssafMSFT good catch it should have been $DroppedDateTime I have since made the correction. actually I have the incorrect command it should be  Get-AzSynapseDroppedSqlPool apologies I was conducting two test let me update that and provide an update here once compelted.

Reply:
Hi @WilliamDAssafMSFT tested and validated all the commands. I used Get-AzSynapseDroppedSqlPool  with the client to get the exact drop date to use for the restore. I added that if the workspace is dropped, they can easily recreate it with the same values and the command will work. Tested that as well. Overall, the invocation of "$DroppedDateTime = Get-AzSynapseDroppedSqlPool -ResourceGroupName $ResourceGroupName -WorkspaceName $WorkspaceName -Name $DatabaseName | Select-Object -ExpandProperty DeletionDate" which should only have one value should prevent the errors that the client experienced when they provided an estimated date time prior to preform the deleted database restore.